index.js

```diff
@@ -4,11 +4,22 @@ console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
-
+const newHeader = document.querySelector("#header")
+console.log(newHeader.style.color = "blue")
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+const updatedTravelerImg = document.querySelector("div#profile img")
+updatedTravelerImg.src = "pics/raffy.jpg"
+updatedTravelerImg.alt = "Raffy's picture"
+
+const updatedTravelerName = document.querySelector("div#profile h2")
+updatedTravelerName.textContent = "Carlos"
+
+const updatedTravelerNickname = document.querySelector("div#profile em")
+updatedTravelerNickname.textContent = "Chuck"
+
+console.log('This is what the traveler object looks like: ', traveler)
 
 
 /***** Deliverable 4 *****/
```

Comment:
here's some hints w/ deliverable 4
You need to create an li for each animal sighting post and the parent should be the animalsUl. You have several options. Think about what we'd need to accomplish this! (hint, innerHTML can help you here.)

**Here's a hint w/ deliverable 5**

Check out [this reference](https://www.w3schools.com/cssref/css_selectors.asp) for info on CSS selectors and look into the `remove` method. How can we find the element we need to remove?